carma/launch/drivers.launch.py

```diff
@@ -47,6 +48,13 @@ def generate_launch_description():
         description = "True if simulation mode is on"
     )
 
+    use_mock_controller = LaunchConfiguration('use_mock_controller')
```

Comment:
I think if we are calling this node, we expect to use the node, so could it be unnecessary to have this flag?

Reply:
I agree, in my story it was mentioned call mock driver if an argument is set. You let me know if I should remove it.

Reply:
removed